kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/cases/CaseRuntimeDataServiceIntegrationTest.java

```diff
@@ -554,6 +554,35 @@ public void testGetCaseInstancesWithData() {
         assertEquals("first case started", instance.getCaseFile().getData().get("s"));
     }
 
+    @Test
+    public void testByPassAuthCaseTasksAssignedAsPotentialOwner() throws Exception {
+        String caseId = startUserTaskCase(USER_YODA, USER_JOHN);
+        assertNotNull(caseId);
+        assertTrue(caseId.startsWith(CASE_HR_ID_PREFIX));
+        try {
+            List<TaskSummary> tasks = caseClient.findCaseTasksAssignedAsPotentialOwner(caseId, USER_YODA, 0, 10);
```

Comment:
As the purpose of this test is to verify that the task lists are returned when sending an empty user, I would change that to (so we make sure the tasks belonging to user Yoda are fetched):

```suggestion
            List<TaskSummary> tasks = caseClient.findCaseTasksAssignedAsPotentialOwner(caseId, "", 0, 10);
```